packages/sdk/wallets/wallet-ledger/patches/@ledgerhq+hw-app-eth+6.36.1.patch

```diff
@@ -25,5 +25,17 @@ index 027089b..92f7d03 100644
 +        }
 +    }
      else {
-         // Legacy tx
+-        // Legacy tx
++        // Celo-Legacy tx
          decodedTx = {
+-            data: rlpDecoded[5],
+-            to: rlpDecoded[3],
+-            // Default to 1 for non EIP 155 txs
+-            chainId: rlpTx.length > 6 ? rlpTx[6] : Buffer.from("0x01", "hex"),
++          data: rlpDecoded[8],
++          to: rlpDecoded[6],
++          // Default to 1 for non EIP 155 txs
++          chainId: rlpTx.length > 9 ? rlpTx[9] : Buffer.from("0x01", "hex"),
```

Comment:
is the buffer right here? it looks like setting chain id to 1.

Reply:
It was like that before but I could remove it as we know celo-legacy encoded rlp always have more than 9 fields